packages/IMAPClient-UI.package/ICFolderDialog.class/instance/buildWith..st

```diff
@@ -83,6 +83,15 @@ buildWith: builder
 		autoDeselect: false;
 		columns: {
 			[:listMorph | self emphasizeUnseenMessages: listMorph filteredItems.
+				listMorph filteredItems
```

Comment:
Why not move this into it's own method like in line 85?

Reply:
Done in 86d2b31.